bvm/ballerina-config/src/main/java/org/ballerinalang/config/ConfigRegistry.java

```diff
@@ -281,7 +281,7 @@ public double getAsFloat(String key) {
                 }
                 return (Double) value;
             } catch (ClassCastException | NumberFormatException e) {
-                throw new IllegalArgumentException(key + " does not map to a valid float");
+                throw new IllegalArgumentException("The config key '" + key + "' does not map to a valid 'float'");
```

Comment:
Shouldn't that error message start with lower cases?

Reply:
We haven't followed one convention for error messages at the moment. Checked some of the panics being created and it has a mix of upper and lower case first letters. Will check with the others as well and see.

Reply:
Changed it to lower case with 27826ad